libs/flux/build.go

```diff
@@ -263,6 +269,31 @@ func downloadModule(logger *zap.Logger) (module.Version, string, error) {
 }
 
 func getVersion(dir string, logger *zap.Logger) (string, error) {
+	if v, err := getVersionFromPath(dir); err != nil {
+		logger.Info("Could not determine version from base path", zap.Error(err))
+	} else {
+		return v, nil
+	}
+
+	if v, err := getVersionFromGit(dir, logger); err != nil {
+		logger.Info("Could not determine version from git data", zap.Error(err))
+	} else {
+		return v, nil
+	}
+	logger.Info("Using default version")
+	return "", errors.New("unable to determine flux repository version")
+}
+
+func getVersionFromPath(dir string) (string, error) {
```

Comment:
So this will succeed if we are building libflux from the module cache (or the Go cache?  I get them confused), right?

Reply:
Yes.